xbmc/guilib/Shader.h

```diff
@@ -32,11 +32,17 @@ namespace Shaders {
     virtual bool InsertSource(const std::string& filename, const std::string& loc);
     bool OK() const { return m_compiled; }
 
+    std::string GetName() { return m_filename; }
+    std::string GetSource();
```

Comment:
Oversaw this last time: Nothing about this indicates that it will line-number the source. Either include it in the function name or as last resort add doxy.

Reply:
Or move the line-numbering to `StringUtils` even?